system/kube-monitoring/charts/prometheus-collector/aggregation.rules

```diff
@@ -1,11 +1,11 @@
 aggregated:container_cpu_system_seconds_total{} = sum(container_cpu_system_seconds_total) by (id) keep_common
 aggregated:container_cpu_usage_seconds_total{} = sum(container_cpu_usage_seconds_total) by (id) keep_common
 aggregated:container_cpu_user_seconds_total{} = sum(container_cpu_user_seconds_total) by (id) keep_common
-aggregated:swift_cluster_storage_used_percent_gauge_average{} = avg(swift_cluster_storage_used_percent_gauge) by (kubernetes_namespace, system, component)
+aggregated:swift_cluster_storage_used_percent_gauge_average{} = avg(swift_cluster_storage_used_percent_gauge) by (kubernetes_namespace, system, component) OR avg(swift_cluster_storage_used_percent) by (kubernetes_namespace, system, component)
```

Comment:
@auhlig: what do you think about that `OR` in the aggr rule?

Reply:
Sorry for missing that PR @reimannf. Looks good.